lambda/executor.js

```diff
@@ -145,6 +145,11 @@ const computeStatistics = (baseCost, results, value, discardTopBottom) => {
     const averageDuration = utils.computeAverageDuration(durations, discardTopBottom);
     console.log('Average duration: ', averageDuration);
 
+    const initDurations = utils.parseLogAndExtractInitDurations(results);
+
+    const [initDuration] = initDurations;
```

Comment:
The assumption is that there's always only one cold start, right? This will change a bit when #177 is merged.

Reply:
Sure, we could use the same `computeAverageDuration` to get the average init duration, passing `discardTopBottom` as zero.